README.md

```diff
@@ -215,3 +215,4 @@ Following table lists all the parameters supported by the latest MarkLogic Helm
 4. The latest released version of fluent/fluent-bit:2.2.2 has known security vulnerabilities with respect to libcom-err2 CVE-2022-1304, libgcrypt20 CVE-2021-33560, libgnutls30 CVE-2024-0567, libldap-2.4-2 CVE-2023-2953, libzstd1 CVE-2022-4899, zlib1g CVE-2023-45853. These libraries are included in the Debian base image but, to-date, no fixes have been made available. For libpq5 CVE-2024-0985, we wait for a future upgrade of the fluent-bit image to include the fix. We will provide updates and mitigation strategies as soon as more information becomes available.
 5. The latest released version of redhat/ubi9:9.3 has known security vulnerabilities with respect to setuptools GHSA-r9hx-vwmv-q579. We wait for a future upgrade of the redhad ubi image to include the fix.
 6. The security context “allowPrivilegeEscalation” is set to TRUE by default in values.yaml file and cannot be changed to run the current MarkLogic container. Work is in progress to run MarkLogic container in "rootless" mode.
+7. You can find the known issues and limitations for the MarkLogic Server image by visiting the following link and checking out the "Known Issues and Limitations" section: https://github.com/marklogic/marklogic-docker?tab=readme-ov-file#Known-Issues-and-Limitations
```

Comment:
You can find the known issues and limitations for the MarkLogic Server image by visiting the following link and checking out the "Known Issues and Limitations" section: https://github.com/marklogic/marklogic-docker?tab=readme-ov-file#Known-Issues-and-Limitations

**If below looks better?** 
Known Issues and Limitations for the MarkLogic Server Docker image can be viewed using the link: https://github.com/marklogic/marklogic-docker?tab=readme-ov-file#Known-Issues-and-Limitations

Reply:
Sure. Will push the change